web/libs/editor/tests/integration/e2e/sync/audio_video_paragraphs.cy.ts

```diff
@@ -182,6 +182,10 @@ describe("Sync: Video Paragraphs", () => {
       });
     });
 
+    // Get more time to ckeck play/pause before the end of the file
+    AudioView.clickAt(100, 0);
```

Comment:
It had not enough time to react to playing when we started playing almost at the end